src/main/services/features/dashboard/claimSummary/latestUpdate/latestUpdateContentBuilder.ts

```diff
@@ -1,14 +1,254 @@
 import {Claim} from 'models/claim';
-import {ClaimSummarySection} from 'form/models/claimSummarySection';
+import {
+  ClaimSummarySection, ClaimSummaryType,
+} from 'form/models/claimSummarySection';
 import {
   getNotPastResponseDeadlineContent,
   getPastResponseDeadlineContent,
   getRespondToClaimLink,
   getResponseNotSubmittedTitle,
 } from './latestUpdateContent/responseToClaimSection';
+import {ClaimResponseStatus} from 'models/claimResponseStatus';
+import {
+  getAmount,
+  getFirstRepaymentDate,
+  getPaymentAmount,
+  getPaymentDate,
+  getRepaymentFrequency,
+} from 'common/utils/repaymentUtils';
+import {DocumentUri} from 'models/document/documentType';
+import {CASE_DOCUMENT_DOWNLOAD_URL, CITIZEN_CONTACT_THEM_URL} from 'routes/urls';
+import {formatDateToFullDate} from 'common/utils/dateUtils';
+import {getLanguage} from 'modules/i18n/languageService';
+import currencyFormat from 'common/utils/currencyFormat';
+
+const PAGES_LATEST_UPDATE_CONTENT = 'PAGES.LATEST_UPDATE_CONTENT.';
+
+export class LastUpdateSectionBuilder {
+  _claimSummarySections: ClaimSummarySection[] = [];
+
+  addTitle(title: string, variables?: any) {
+    const titleSection = ({
+      type: ClaimSummaryType.TITLE,
+      data: {
+        text: title,
+        variables: variables,
+      },
+    });
+    this._claimSummarySections.push(titleSection);
+    return this;
+  }
+
+  addParagraph(text: string, variables?: any) {
+    const paragraphSection = ({
+      type: ClaimSummaryType.PARAGRAPH,
+      data: {
+        text: text,
+        variables: variables,
+      },
+    });
+    this._claimSummarySections.push(paragraphSection);
+    return this;
+  }
+
+  addContactLink(text: string, claimId: string, variables?: any, textAfter?: string) {
+    const linkSection = ({
+      type: ClaimSummaryType.LINK,
+      data: {
+        text: text,
+        variables: variables,
+        href: CITIZEN_CONTACT_THEM_URL.replace(':id', claimId),
+        textAfter: textAfter,
+      },
+    });
+
+    this._claimSummarySections.push(linkSection);
+    return this;
+  }
+  addResponseDocumentLink(text: string, claimId: string, variables?: any, textAfter?: string) {
+    const linkSection = ({
+      type: ClaimSummaryType.LINK,
+      data: {
+        text: text,
+        variables: variables,
+        href: CASE_DOCUMENT_DOWNLOAD_URL.replace(':id', claimId).replace(':documentType', DocumentUri.SEALED_CLAIM),
+        textAfter: textAfter,
+      },
+    });
+
+    this._claimSummarySections.push(linkSection);
+    return this;
+  }
+
+  build() {
+    return this._claimSummarySections;
+  }
+}
+
+function getPaPaidPayInstallmentItems(claim: Claim) {
```

Comment:
Maybe call this getPartAdmitPayInstallmentItems or similar? Assuming Pa stands for Part Admit here

```suggestion
function getPartAdmitPayInstallmentItems(claim: Claim) {
```